### PR TITLE
[14.0][ADD] shopinvader_product_manufactured_for

### PR DESCRIPTION
**Pull request by kv1612**
Glue module between `sale_product_manufactured_for` and Shopinvader.

### CODE REVIEW

File: shopinvader_product_manufactured_for/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+----
+This module has been proposed on odoo-shopinvader repository by this
+`PR <https://github.com/shopinvader/odoo-shopinvader/pull/722>`_.
+But refused because it contains Odoo business logic.
+
+Roadmap
+-------
+
+Split this into two module one for the Odoo business logic in OCA and another
+one implementing the export to Shopinvader.
```
Comment (simahawk) [inline shopinvader_product_manufactured_for/readme/DESCRIPTION.rst]:
```suggestion
Glue module between `sale_product_manufactured_for` and Shopinvader.
```

**Review by mmequignon — APPROVED** [review]

**Review by simahawk — CHANGES_REQUESTED** [review]

**Comment by kv1612** [comment]
Hi @simahawk, I have just updated DESCRIPTION.rst. Thank you !

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
@kv1612 pls rebase and check pre-commit

**Comment by kv1612** [comment]
Hi @simahawk, this PR has been updated.

**Pull request merged**